css/style.css

```diff
@@ -0,0 +1,403 @@
+html,
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+ul {
+  list-style: none;
+  display: block;
+  margin-block-start: 0;
+  margin-block-end: 0;
+  margin-inline-start: 0;
+  margin-inline-end: 0;
+  padding-inline-start: 0;
+}
+
+figure {
+  display: block;
+  margin-block-start: 0;
+  margin-block-end: 0;
+  margin-inline-start: 0;
+  margin-inline-end: 0;
+}
+
+body {
+  top: 50px;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+}
+
+img {
+  max-width: 100%;
+}
+
+nav {
```

Comment:
:bulb: Let's add a `z-index` property here. Since we want the navbar to always be on top, it will be better to set a property that would ensure that. `z-index` do this for positioned elements. 
```suggestion
nav {
   z-index: 1;
```

![z-indexnyt](https://user-images.githubusercontent.com/36057474/64207336-57800480-ce94-11e9-81a8-1e5cc68b400a.gif)

Learn more [here](https://developer.mozilla.org/en-US/docs/Web/CSS/z-index)